src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsModel.ts

```diff
@@ -60,6 +60,8 @@ export class InlineCompletionsModel extends Disposable {
 		private readonly _suggestPreviewMode: IObservable<'prefix' | 'subword' | 'subwordSmart'>,
 		private readonly _inlineSuggestMode: IObservable<'prefix' | 'subword' | 'subwordSmart'>,
 		private readonly _enabled: IObservable<boolean>,
+		private readonly _isScreenReaderEnabled: IObservable<boolean>,
+		private readonly _editorDictationInProgress: IObservable<boolean>,
```

Comment:
@hediet is there a way to combine observables to prevent having to pass these in? 🤔

Reply:
You can combine observables with `derived(owner, reader => ...)`. Use `.read(reader)` to read the values from other observables such that the derived recomputes when the read observables change.